frontend/pages/mediaSizes.tsx

```diff
@@ -0,0 +1,35 @@
+// from https://jsramblings.com/how-to-use-media-queries-with-styled-components/
+// thx Stefan <3 very cool
+import React, { ReactElement, useEffect, useState } from 'react';
+
```

Comment:
this file shouldn't be in the pages directory

Pages directory is specifically for files that export components that automatically make routes in the site

since this file is in pages, and doesn't export a react component, the build is failing (in Vercel)
to fix, put this file into another utility folder

side note:
if this did export a react component, then knowyouroptions.com/mediaSizes would show that component